8-nodes-block-4-2/rolling_static.yaml

```diff
@@ -0,0 +1,6 @@
+- hosts: ydb
+  become: true
+  tasks:
+    - name: ydb storage nodes rolling restart
+      ansible.builtin.import_role:
+        name: ydb_platform.ydb.ydbd_rolling_static
```

Comment:
This change should follow this discussion https://github.com/ydb-platform/ydb-ansible-examples/pull/5/files#r1590657020